swing-lib/src/main/java/org/triplea/swing/JTableBuilder.java

```diff
@@ -76,6 +76,12 @@ public static void addRows(final JTable table, final List<List<String>> rows) {
     rows.forEach(row -> model.addRow(row.toArray(new String[0])));
   }
 
+  public static void setData(final JTable table, final List<List<String>> rows) {
```

Comment:
This is needed for when we click 'refresh' on the events tab. That is a case where we want to overwrite existing JTable data.